manifestpy/mapmain.py

```diff
@@ -0,0 +1,33 @@
+def mapMainManifest(readfile, wtype):
+
+    mainOut = '{"@context":"http://schema.org",'
+    mainOut += '"@type":"' + wtype + '",'
+    if 'label' in readfile.keys(): mainOut += '"name:":"' + readfile['label'] + '",'
```

Comment:
For consistency with other Python code we're writing, you may want to install the Flake 8 linter.  It would recommend you separate the if on a different line from the rest of the code, like this:

if 'label' in readfile.keys():
  mainOut...